### PR TITLE
Use new x icon for notification close

### PR DESCRIPTION
**Pull request by jingpei**

### CODE REVIEW

**Comment by jingpei** [comment]
Not exactly sure if the package-lock file should have been committed, it was updated after I npm installed or possibly after npm linking my local gringotts

File: templates/notification.hbs [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+<button type="button" class="close" data-dismiss="alert" aria-hidden="true">
+	{{icon 'misc-x'}}
```
Comment (eugenet8k) [inline templates/notification.hbs]: I think we should use two space gap here, also maybe moving button after label will be semantically more correct?

File: package-lock.json [hunk 2]
```diff
@@ -3578,6 +4478,13 @@
       "integrity": "sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=",
       "dev": true
     },
+    "nan": {
```
Comment (eugenet8k) [inline package-lock.json]: this is strange, it seems you somehow introduce new dependencies. Or some of the npm dependencies got updated withing last hour..

Comment (jingpei) [reply package-lock.json]: @bzalasky says to leave it since it does sometimes update if one of our locked dependencies updates one of its dependencies

Comment (eugenet8k) [reply package-lock.json]: @jingpei what version of NodeJs you used while installing npm dependencies for gringotts?

Comment (jingpei) [reply package-lock.json]: @eugenet8k I was using 6.1.0

**Pull request merged**